main/options.c

```diff
@@ -502,6 +503,9 @@ static const char *const Features [] = {
 	"json",
 	"interactive",
 #endif
+#ifdef HAVE_SECCOMP
+        "secure",
```

Comment:
this is what c-guess came up with. Maybe it's something wrong with the binding of TAB ?

(BTW, I can recommend https://clang.llvm.org/docs/ClangFormat.html as a solution to formatting dilemmas.)